src/data/arithm.rs

```diff
@@ -457,6 +457,50 @@ impl Number {
             _ => Some(val),
         }
     }
+
+    /// Adds or removes negative sign to the number (negates negative or positive number, depending
+    /// on the method argument value)
+    ///
+    /// # Panics
```

Comment:
We need to change this section from "Panics" to "Return" describing these conditions as such when the function returns `None`